django_netjsonconfig/static/django-netjsonconfig/js/widget.js

```diff
@@ -67,13 +67,44 @@
         return editor;
     };
 
+    var evaluateVars = function(data, context){
+        var pattern = new RegExp('\{\{.*?\}\}');
+                if (typeof data === 'object'){
```

Comment:
the indentation of this line looks wrong, it looks to me that the entire block shoud be de-indented